libsplinter/Cargo.toml

```diff
@@ -211,7 +211,7 @@ service-timer =[
   "defered-send",
   "runtime-service",
   "service-message-sender-factory",
-  "service-timer-alarm",
```

Comment:
Shouldn't both of these features exist? Also, this doesn't seem to be in the correct commit.

Reply:
This is updating which feature the `service-timer` requires. Which is required as of this commit. The  "service-timer-alarm-factory" pulls in "service-timer-alarm". No features are actually removed here.